HACKING.md

```diff
@@ -9,6 +9,15 @@ In order to build itself, Dune uses an OCaml script
 single `boot.ml` file. This file is built using `ocamlopt` or `ocamlc`
 and used to build everything else.
 
+To get the development version bootstraped and installed, run the following
+while in the project's root directory:
+
+```sh
+$ make boot.exe # create the dune.exe file
+$ ./boot.exe    # bootstrap the dune build
```

Comment:
I think that the two first commands can be replaced by just `make`. Is there anything missing when you run this?

Reply:
Oh yes, of course! I'll set up a clean build to be certain, but I'm sure you're right.